commands/license_info

```diff
@@ -27,6 +24,7 @@ EOF
 table_format=true
```

Comment:
- I'd suggest to also set table_format to false here
- in the else block of `if [ $# -gt 1 ]; then` (line 39), you can set it to true (this should catch the default case where the user did not explicitly specify an output format)

Reply:
I put table_format=true before checking the argument to make sure that if users run only c8y oee license_info then it will print automatically in table format. If I put it in any place inside the argument check section, then it requires to have a specific setup to have table format.